src/extension.ts

```diff
@@ -37,7 +38,13 @@ export function activate(context: ExtensionContext) {
 
   context.subscriptions.push(createTerminateDebugListener());
 
+  /**
+   * Register Yaml Schema support to manage application.yaml
+   */
+  const yamlSchemaCache = registerYamlSchemaSupport();
+
   connectToLS(context).then(() => {
+    yamlSchemaCache.then(cache => cache.languageClient = languageClient);
```

Comment:
Please check if cache is not undefined first `if (cache) {cache.languageClient = languageClient}`

Reply:
done